python-sdk/tests/files/locations/test_location_base.py

```diff
@@ -142,3 +138,9 @@ def test_location_eq(loc_1, loc_2, equality):
 def test_location_hash():
     """Test that hashing works"""
     assert isinstance(hash(LocalLocation("/tmp/file_a.csv")), int)
+
+
+def test_invalid_conn_id_with_file_path():
+    """Raise a value when the connection types doesn't match the path"""
+    with pytest.raises(ValueError):
+        GCSLocation("gs://tmp/file_a.csv", conn_id="aws_default")
```

Comment:
fyi: you can pass a `match` param to `pytest.raises` to check for the error message:

Example:

```python
import pytest


def myfunc():
    raise ValueError("Exception 123 raised")


def test_match():
    with pytest.raises(ValueError, match=r".* 123 .*"):
        myfunc()
```

Reply:
From https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions

Reply:
Thanks. [Done](https://github.com/astronomer/astro-sdk/pull/1478/commits/50bd36d6d59208d166c01b6c00eec416cebc6e15)